ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -402,7 +402,18 @@ tests:
       TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;Serial;Disruptive;~DisconnectedOnly&;~HyperShiftMGMT&
     test:
     - chain: openshift-e2e-test-qe-destructive
-    workflow: cucushift-installer-rehearse-aws-ipi-ovn-ingress-nlb
+    workflow: cucushift-installer-rehearse-aws-ipi-ingress-lbtype-nlb
+- as: e2e-aws-ipi-ovn-ingress-lbtype-classic-p2
```

Comment:
Since the new workflow does not contain `-ovn`, we'd better remove it from job name, too.

Reply:
Good catch! thank you.
`-ovn` should not be there.